DuckDuckGo/LaunchOptionsHandler.swift

```diff
@@ -32,12 +31,8 @@ public final class LaunchOptionsHandler {
         self.userDefaults = userDefaults
     }
 
-    public var isUITesting: Bool {
-        launchArguments.contains(Self.isUITesting)
-    }
-
     public var isOnboardingCompleted: Bool {
-        userDefaults.string(forKey: Self.isOnboardingcompleted) == "true"
+        ProcessInfo.processInfo.environment["ONBOARDING"] == "false" || userDefaults.string(forKey: Self.isOnboardingcompleted) == "true"
```

Comment:
We should inject `ProcessInfo.processInfo.environment` so we can test the logic.

`public init(launchArguments: [String] = ProcessInfo.processInfo.arguments, environment: [String: String] =  ProcessInfo.processInfo.environment, userDefaults: UserDefaults = .app) {
   self.launchArguments = launchArguments
   self.environment = environment
   self.userDefaults = userDefaults
}

```suggestion
        environment["ONBOARDING"] == "false" || userDefaults.string(forKey: Self.isOnboardingcompleted) == "true"
```